src/patternfly/components/Button/styles.scss

```diff
@@ -290,6 +312,71 @@
     }
   }
 
+  // Link and Action Buttons
+  &.pf-m-link,
+  &.pf-m-action {
+    background-color: transparent;
+    box-shadow: none;
+
+    &:hover,
+    &.pf-m-hover,
+    &:active,
+    &.pf-m-active,
+    &:focus,
+    &.pf-m-focus {
+      background-color: transparent;
+      box-shadow: none;
+    }
+  }
+
+  // Link Buttons
+  &.pf-m-link {
+    display: inline-flex;
+    padding: var(--pf-c-button--m-link--PaddingTop) var(--pf-c-button--m-link--PaddingRight) var(--pf-c-button--m-link--PaddingBottom) var(--pf-c-button--m-link--PaddingLeft);
```

Comment:
Please don't change the padding since this is adding the click area for touch devices.

Reply:
If button is the first to appear in a button group or on a new line below another element, including the default left/right padding creates an offset.

<img width="531" alt="screen shot 2018-06-05 at 1 03 19 pm" src="https://user-images.githubusercontent.com/5385435/40991247-286c659c-68c1-11e8-800c-269eed0f44e7.png">

<img width="483" alt="screen shot 2018-06-05 at 1 03 29 pm" src="https://user-images.githubusercontent.com/5385435/40991253-2bfc8160-68c1-11e8-859b-1bbc3b747115.png">

Reply:
@andresgalante should we use the default padding with negative margin button wrapper to accommodate this use case^^?